superset-frontend/src/dashboard/components/nativeFilters/FilterBar/Header/index.tsx

```diff
@@ -116,7 +116,7 @@ const Header: FC<HeaderProps> = ({ toggleFiltersBar }) => {
     <Wrapper>
       <TitleArea>
         <span>{t('Filters')}</span>
-        {canSetHorizontalFilterBar && <FilterBarOrientationSelect />}
+        {canSetHorizontalFilterBar && <FilterBarSettings />}
```

Comment:
The gear icon is not visible if `HORIZONTAL_FILTER_BAR` is disabled.

Reply:
Great catch